Lib/Rabbit4000/tcpip/DMAETH100.LIB

```diff
@@ -808,34 +817,33 @@ dma_rcv_isr::
    jr    nz,.reset_rx_port ; 5
 
    ; Check if only status rcv'd with no packet data
+   ; This is the catch all for errors with R6000
 	ld		hl,(ix+[_decb]+rcvlen)
    test	hl
    jr		z,.test_rx_error
 
    ; Check for valid Rcv Packet
-	ld		a,(ix+[_decb]+rcvlsbstat)
+   ; Don't know where this is documented for R6000?

```

Comment:
The 10/100 Ethernet section of the 6000 manual is similar to the 5000, and from my cursory review I think the 6000 uses the same 10/100 MAC code but adds a PHY.  I don't see any reason that those registers would have changed between the 5000 and 6000 CPUs.